src/sql/base/browser/ui/table/plugins/rowMoveManager.plugin.ts

```diff
@@ -0,0 +1,169 @@
+// Adopted and converted to typescript from https://github.com/mleibman/SlickGrid/blob/gh-pages/plugins/slick.rowmovemanager.js
```

Comment:
add our common file header

Reply:
the plugins are not owned by us and thus dont have copyright headers